assets/scss/base/_helpers.scss

```diff
@@ -373,3 +373,18 @@
   max-width: 125px;
 }
 
+/*
+Equally spaced out container element
+
+Experimental: Equal top and bottom spacing for container element
+
+Markup:
+<div class="{{modifier_class}}">Content</div>
```

Comment:
no need for `{{modifier_class}}` here you can just place the class `equal-spacing` straight in the example